generated/python/proto-google-cloud-datastore-v1/setup.py

```diff
@@ -11,13 +11,19 @@
 
 install_requires = [
   'oauth2client>=2.0.0, <4.0dev',
-  'grpcio>=1.0.2rc0, <2.0dev',
-  'googleapis-common-protos[grpc]>=1.5.0, <2.0dev'
+  'googleapis-common-protos>=1.5.0, <2.0dev',
 ]
 
+extras_require = {
+  'grpc': [
+    'googleapis-common-protos[grpc]>=1.5.0, <2.0dev',
+    'grpcio>=1.0.2, <2.0dev',
+  ],
+}
+
 setuptools.setup(
-  name='grpc-google-cloud-datastore-v1',
-  version='0.14.0',
+  name='proto-google-cloud-datastore-v1',
+  version='0.15.0',
```

Comment:
As soon as we declare Beta on the Python codegen, we can set all packages to be `0.9x.x` and `4 - Beta`. Main things we need are automated unit and smoke tests, and GAPIC / gRPC docs published for all packages.

We also have talked about allowing version overrides in the GAPIC config, since we may not always want to move the packages together in sync.